sambaplayersdk/src/main/java/com/sambatech/player/model/SambaMediaRequest.java

```diff
@@ -14,6 +14,7 @@
 	public String liveChannelId;
 	public String streamName;
 	public String streamUrl;
+	public boolean isStreamAudio;

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make isStreamAudio a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.qa.sambatech.com/sonar/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)